doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function does not work on AIX versions older than 7.1. It can still
+be called but will return `UV_ENOSYS`.
+
```

Comment:
Are you saying that older for you means later? 

> it only worked on version 7.1 and newer

>does not work on AIX versions older than 7.1

These two phrases seem identical to me.